src/content/docs/iast/applications.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: IAST Tests & Applications
+tags:
+    - IAST
+    - New Relic IAST
+    - Applications
+metaDescription: Find out the total number of exploitable applications and safe applications you have.
+freshnessValidatedDate: never
+---
+
+import iastCoverageSummary from 'images/iast_screenshot-crop_coverage-summary.webp'
+
+import iastEnableIast from 'images/iast_screenshot-crop_enable-iast.webp'
+
+
+The applications page informs you about the total number of found vulnerabilities in your tested applications and the number of untested applications you have. You can always add untested applications to the New Relic IAST capability. Each application test is a build of an application. Each time your pre-production environment deploys a new build and exercises a new test, it will show up with that application name. For each test run we show the application name, account, status, exploit count, test efficiency, and the last time the application was tested. You can sort by application name to see all test runs of an application together to quickly compare their status.
+
+## Details and remediation of an application [#remediation-app]
+
+Check out all the issues detected with IAS testing on your applications.
+
+To see the details and the remediation instructions for an application:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** and click **Tests & Applications** on the left navigation pane.
+
+2. Select an application. You can order all the columns.
+
+<img
+    title="Details of an application"
+    alt="Details of an application"
+    src={iastCoverageSummary}
+/>
+
+## Enable New Relic IAST on an untested application [#enable-iast]
+
+To enable IAST on an untested application:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** and click **Tests & Applications** on the left navigation pane.
+
+2. Click on the **All applications** tab and check the **IAST supported** column to make sure the application is supported.
+
+3. Select the application you want to enable IAST.
```

Comment:
I think it should be "you want to enable IAST for." or something like that? Unsure though